compass/landice/tests/ismip6_run_ais/streams.landice.template

```diff
@@ -0,0 +1,159 @@
+        <var name="regionCellMasks"/>
+
+</stream>
+
+<stream name="ismip6"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_TF_forcing }}"
+    input_interval="0001-00-00_00:00:00"
+    record_interval="0001-00-00_00:00:00"
+                  reference_time="1995-01-01_00:00:00"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_3dThermalForcing"/>
+        <var name="ismip6shelfMelt_zOcean"/>
+</stream>
+
+<stream name="ismip6_2"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_basal_param }}"
+        input_interval="initial_only"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_deltaT"/>
+        <var name="ismip6shelfMelt_basin"/>
+        <var name="ismip6shelfMelt_gamma0"/>
+</stream>
+
+<stream name="ismip6_3"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_shelf_melt_offset }}"
+        input_interval="initial_only"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_offset"/>
+</stream>
+
+<stream name="ismip6_smb"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_SMB_forcing }}"
+    input_interval="0001-00-00_00:00:00"
+    record_interval="0001-00-00_00:00:00"
+                  reference_time="1995-01-01_00:00:00"
+        runtime_format="single_file">
+        <var name="sfcMassBal"/>
+</stream>
+
+<stream name="output"
+        type="output"
+        filename_template="output.nc"
+        output_interval="0001-00-00_00:00:00"
+        reference_time="1995-01-01_00:00:00"
+        clobber_mode="truncate">
+
+    <stream name="basicmesh"/>
+    <var name="xtime"/>
+    <var name="thickness"/>
+    <var name="upperSurface"/>
+    <var name="lowerSurface"/>
+    <var name="bedTopography"/>
+    <var name="uReconstructX"/>
+    <var name="uReconstructY"/>
+    <var name="xvelmean"/>
+    <var name="yvelmean"/>
+    <var name="surfaceTemperature"/>
+    <var name="basalTemperature">
+    <var name="daysSinceStart"/>
+    <var name="surfaceSpeed"/>
+    <var name="cellMask"/>
+    <var name="edgeMask"/>
+
+</stream>
+
+<stream name="outputFL"
+         type="output"
+         filename_template="FL_outputs/outputFL.$Y.nc"
+         filename_interval="0000-00-00_01:00:00"
+         precision="double"
+                  reference_time="1995-01-01_00:00:00"
+         clobber_mode="truncate"
+         output_interval="0001-00-01_00:00:00">
+
+    <var name="xtime"/>
+    <var name="deltat"/>
+    <var name="daysSinceStart"/>
+    <var name="basalHeatFlux"/>
+    <var name="sfcMassBalApplied"/>
+    <var name="groundedBasalMassBalApplied"/>
+    <var name="floatingBasalMassBalApplied"/>
+    <var name="dHdt"/>
+    <var name="calvingThickness"/>
+    <var name="fluxAcrossGroundingLine"/>
+    <var name="edgeMask"/>
+    <var name="cellMask"/>
+</stream>
+
```

Comment:
We might also want the 2 streams we have been using in our testing on Badger to appear.  We could then comment them out if we don't want them.  Let's discuss.